integration-cli/docker_cli_run_unix_test.go

```diff
@@ -295,7 +295,10 @@ func (s *DockerSuite) TestRunEchoStdoutWithMemoryLimit(c *check.C) {
 	}
 }
 
-// should run without memory swap
+// should run without memory swap limit
```

Comment:
Can we start the comment with `TestRunWithoutMemoryswapLimit should ...`, and also leave a space between `//` and words in the lines before? That way we're more consistent with go's lint :smile_cat: